boards/arm/mps2_an385/mps2_an385.dts

```diff
@@ -129,7 +129,7 @@
 			interrupts = <17 3>;
 		};
 
-		eth0: eth@0 {
+		eth0: eth@40200000 {
 			/* Linux has "smsc,lan9115" */
 			compatible = "smsc,lan9220";
 			/* Such a big size from memory map in AN385 */
```

Comment:
@ioannisg, @galak: Here it is:

> /* Such a big size from memory map in AN385 */

AN385 is of course an application note from ARM which describes this MPS2 setup.

So as you can see, I thought about this matter, presented with option to either record the official information, or apply my "this can't be true!" speculative "optimizations", you see what choice I made ;-).